src/cloud/env/environments.md

```diff
@@ -16,3 +16,13 @@ You can configure application settings, routes, build and deploy actions, and no
 -  [`.magento/services.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_services.html)—defines the services Magento uses by name and version. For example, this file may include versions of MySQL, PHP extensions, Redis, RabbitMQ, and Elasticsearch. You must open a support ticket to push these changes to Pro plan Staging and Production environments.
 
 When you push code changes, the active environment provisions container updates using the YAML configuration files.
+
+## Update Pro Staging and Production environment configuration
+
+For {{site.data.var.ece}} Pro Staging and Production environments, you can update many configuration options in your local development environment and commit the changes to apply them to these environments. However, you must submit a support ticket to update the following configuration options:
+
+-  Install or update services in the `.magento.services.yaml` file.
```

Comment:
will it be displayed as `.magento/services.yaml`?

Reply:
Fixed that. Thx.